tests/workflows/titan-gc/test_clearlabs.yml

```diff
@@ -0,0 +1,38 @@
+- name: titan_clearlabs
+  command: titan-gc-cli -i titan/clearlabs.json -o titan/clearlabs
+  tags:
+    - titan_gc
+    - titan_clearlabs
+  files:
+    - path: titan/clearlabs/alignments/clearlabs.flagstat.txt
+      md5sum: b5af0b1c2721c004f8f476d9ca540a35
+    - path: titan/clearlabs/alignments/clearlabs.primertrimmed.rg.sorted.bam
+    - path: titan/clearlabs/alignments/clearlabs.primertrimmed.rg.sorted.bam.bai
+    - path: titan/clearlabs/alignments/clearlabs.stats.txt
+    - path: titan/clearlabs/assemblies/clearlabs.medaka.consensus.fasta
+      md5sum: 1d537e1a17fabda77371ba22bdfb3d80
+    - path: titan/clearlabs/cromwell-stderr.txt
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
+    - path: titan/clearlabs/cromwell-stdout.txt
+    - path: titan/clearlabs/dehosted_reads/clearlabs_R1_dehosted.fastq.gz
+    - path: titan/clearlabs/kraken2_reports/clearlabs_dehosted_kraken2_report.txt
+      md5sum: 35841fa2d77ec202c275b1de548b8d98
+    - path: titan/clearlabs/kraken2_reports/clearlabs_kraken2_report.txt
+      md5sum: 35841fa2d77ec202c275b1de548b8d98
+    - path: titan/clearlabs/nextclade/clearlabs.medaka.consensus.nextclade.auspice.json
+      md5sum: ade0ebc3992e167daf1b8ff645bb4f1c
+    - path: titan/clearlabs/nextclade/clearlabs.medaka.consensus.nextclade.json
+      md5sum: 251e589243c60cca9ad7a684f1192500
+    - path: titan/clearlabs/nextclade/clearlabs.medaka.consensus.nextclade.tsv
+      md5sum: eccf29f64fe718246ce35b89f21bf98c
+    - path: titan/clearlabs/pangolin_reports/clearlabs.pangolin_report.csv
```

Comment:
I'm wondering if we want to exclude pangolin and nextclade results since those may change as default images will likely be updates as we make changes to the whole repo

Reply:
I think include them, but remove the MD5sum check. This way we are still testing the file exists

Reply:
Ah, good call. If I could add a Bob Ross emoji to a GitHub comment I would